includes/options/class-amp-options-menu.php

```diff
@@ -473,7 +297,7 @@ public function render_supported_templates() {
 				$checked = (
 					post_type_supports( $post_type->name, AMP_Post_Type_Support::SLUG )
 					||
-					( ! AMP_Options_Manager::is_website_experience_enabled() && in_array( $post_type->name, $supported_post_types, true ) )
```

Comment:
With this change, the `post` post type will be enabled by default in the supported post types.